report.js

```diff
@@ -64,13 +60,18 @@ async function comment(github, context, summary) {
   })
 }
 
-module.exports.process = async ({ github, context, core }) => {
+// writeSummary writes an step job summary based on the metrics file.
+module.exports.writeSummary = async (core) => {
+  // eslint-disable-next-line no-undef
   const metrics = JSON.parse(require('fs').readFileSync(process.env.METRICS, 'utf8'));
```

Comment:
Given that we use the `fs` module can we require it at the beginning of the file?

```js
const fs = require('fs');
```

```suggestion
  const metrics = JSON.parse(fs.readFileSync(process.env.METRICS, 'utf8'));
```